src/mixins/filing-template-mixin.ts

```diff
@@ -200,11 +200,14 @@ export default class FilingTemplateMixin extends DateMixin {
         }
 
         if (this.isTypeBcUlcCompany) {
+          // Add Court Order ONLY when it is required and applied.
           const courtOrder = this.getCourtOrderStep.courtOrder
-          filing.incorporationApplication.courtOrder = {
-            fileNumber: courtOrder.fileNumber,
-            effectOfOrder: courtOrder.hasPlanOfArrangement ? EffectOfOrders.PLAN_OF_ARRANGEMENT : '',
-            hasPlanOfArrangement: courtOrder.hasPlanOfArrangement
+          if (courtOrder.hasPlanOfArrangement || courtOrder.fileNumber) {
```

Comment:
The courtOrder attribute is only populated if the court order number is provided or hasPlanOfArrangement is checked.